manticore/manticore.py

```diff
@@ -628,6 +708,15 @@ def terminate(self):
     #############################################################################
     # Move all the following elsewhere Not all manticores have this
 
+    def add_symbolic_file(self, symbolic_file):
```

Comment:
this may have gotten added back to manticore.py in the merge; it should be in linux.py

Reply:
What about _get_symbol_address(self, symbol) ?

Reply:
Let's leave it under the "Move all the following elsewhere Not all manticores have this FIXME"